new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSystemResOverrides.java

```diff
@@ -0,0 +1,514 @@
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    //determine if the tests are running in Kind cluster. if true use images from Kind registry
+    if (KIND_REPO != null) {
+      String kindRepoImage = KIND_REPO + image.substring(TestConstants.OCR_REGISTRY.length() + 1);
+      logger.info("Using image {0}", kindRepoImage);
+      image = kindRepoImage;
+      isUseSecret = false;
+    } else {
+      // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+      createOCRRepoSecret(domainNamespace);
+    }
+    //create and start WebLogic domain
+    createDomain();
+
+    // build the sitconfig application
+    Path distDir = buildApplication(Paths.get(APP_DIR, "sitconfig"),
+        null, null, "dist", domainNamespace);
+    sitconfigAppPath = Paths.get(distDir.toString(), "sitconfig.war");
+    assertTrue(sitconfigAppPath.toFile().exists(), "Application archive is not available");
+
+    //deploy application to view server configuration
+    deployApplication(clusterName + "," + adminServerName);
+
+  }
+
+  /**
+   * Test JMS and WLDF system resources configurations are overridden dynamically when domain resource
+   * is updated with overridesConfigMap property. After the override verifies the system resources properties
+   * are overridden as per the values set.
+   */
+  @Test
+  @DisplayName("Test JMS and WLDF system resources override")
+  public void testSystemResourceOverride() {
+
+    //store the pod creation timestamps
+    storePodCreationTimestamps();
+
+    List<Path> overrideFiles = new ArrayList<>();
+    overrideFiles.add(
+        Paths.get(RESOURCE_DIR, "configfiles/configoverridesset2/jms-ClusterJmsSystemResource.xml"));
+    overrideFiles.add(Paths.get(RESOURCE_DIR, "configfiles/configoverridesset2/diagnostics-WLDF-MODULE-0.xml"));
+    overrideFiles.add(Paths.get(RESOURCE_DIR, "configfiles/configoverridesset2/version.txt"));
+
+    //create config override map
+    createConfigMapFromFiles(overridecm, overrideFiles, domainNamespace);
+
+    String introspectVersion = assertDoesNotThrow(() -> getNextIntrospectVersion(domainUid, domainNamespace));
+
+    logger.info("patch the domain resource with overridesConfigMap and introspectVersion");
+    String patchStr
+        = "["
+        + "{\"op\": \"add\", \"path\": \"/spec/configuration/overridesConfigMap\", \"value\": \"" + overridecm + "\"},"
+        + "{\"op\": \"add\", \"path\": \"/spec/introspectVersion\", \"value\": \"" + introspectVersion + "\"}"
+        + "]";
+    logger.info("Updating domain configuration using patch string: {0}", patchStr);
+    V1Patch patch = new V1Patch(patchStr);
+    assertTrue(patchDomainCustomResource(domainUid, domainNamespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH),
```

Comment:
The description does not says, we need to bump up introspector version. If so please update the javadoc to detail the steps in the usecase

Reply:
improved the javadoc